src/mesh/NodeDB.cpp

```diff
@@ -308,7 +308,11 @@ void NodeDB::installDefaultConfig(bool preserveKey = false)
 #else
     config.lora.modem_preset = meshtastic_Config_LoRaConfig_ModemPreset_LONG_FAST;
 #endif
+#ifdef USERPREFS_CONFIG_LORA_HOPS
+    config.lora.hop_limit = USERPREFS_CONFIG_LORA_HOPS;
+#else
```

Comment:
Given the history we've had with this setting, i am strongly not a fan of making it bakeable into firmware.

Reply:
It's use case dependent, and is currently manipulated via the`USERPREFS_EVENT_MODE` user prefs for valid reasons. What is the specific concern?

Reply:
People giving out compiled firmware with this set to "7" and marketing it as "meshtastic range boosted special hacker version"

Reply:
I'm not in favor of this either. We're working on making an interface to allow users to custom define userPrefs for custom builds. Allowing users to bake in 7 hop default firmware is like handing out the plans on how to build a mesh killing gun.